src/components/Contracts/ContractItem/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import Card from 'antd/es/card';
+import List from 'antd/es/list';
+import Title from 'antd/es/typography/Title';
+import Paragraph from 'antd/es/typography/Paragraph';
+import Markdown from 'markdown-to-jsx';
+import router from '../../../router';
+import { blockchainSelectors, contractsSelectors } from '../../../redux/selectors';
+import Button from '../../Button/Button';
+import ButtonsWrapper from '../ButtonsWrapper';
+import CopyIconWithAddress from '../../CopyIconWithAddress';
+import { deriveAndHideContractTitle } from './utils';
+
+function ContractItem({
```

Comment:
You removed the feature that contracts are collapsed by default, only showing a few lines, not entire contract in overview of all contracts. Reimplement pls

Reply:
Added back thanks!